src/ResourceManager/Sql/Commands.Sql/VirtualNetworkRule/Cmdlet/RemoveAzureSqlServerVirtualNetworkRule.cs

```diff
@@ -0,0 +1,89 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Sql.VirtualNetworkRule.Cmdlet
+{
+    /// <summary>
+    /// Defines the Remove-AzureRmSqlServerVirtualNetworkRule cmdlet
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "AzureRmSqlServerVirtualNetworkRule", SupportsShouldProcess = true, ConfirmImpact = ConfirmImpact.Medium)]
+    public class RemoveAzureSqlServerVirtualNetworkRule : AzureSqlServerVirtualNetworkRuleCmdletBase
+    {
+        /// <summary>
+        /// Gets or sets the name of the VirtualNetwork rule to remove
+        /// </summary>
+        [Parameter(Mandatory = true,
+            HelpMessage = "Azure Sql Server Virtual Network Rule name")]
+        [ValidateNotNullOrEmpty]
+        public string VirtualNetworkRuleName { get; set; }
+
+        /// <summary>
+        /// Defines whether it is ok to skip the requesting of rule removal confirmation
+        /// </summary>
+        [Parameter(HelpMessage = "Skip confirmation message for performing the action")]
+        public SwitchParameter Force { get; set; }
```

Comment:
@kisantia this `Force` parameter should not be needed unless there is a special circumstance where you want to prompt the user to confirm their actions. I would take a look at Adam's comment below to see if this holds true.